docs/guide.md

```diff
@@ -51,7 +51,16 @@ packages. SciCookie support the following:
   PEP 517 compilation backend, PEP 621 project metadata, a flexible and
   powerful plugin system. It also offers, among other things, versatile
   user scripting, PyPI integration and version management.
-
+- [**Hatchling**](https://hatch.pypa.io): It's a is a PEP 517/PEP 660 compatible
+  build backend used by Hatch, a modern, extensible Python project manager.
+  It provides a standardized build system with reproducible builds by default,
+  robust environment management with support for custom scripts, easy publishing
+  to PyPI or other indexes, version management, and configurable project generation
+  with sane defaults.Hatchling ensures that your builds are reproducible, so you
+  can be confident that they will always produce the same results. It also helps
+  you manage your Python environments, so you can be sure that your projects have
+  the correct dependencies.
```

Comment:
```suggestion
- [**Hatchling**](https://hatch.pypa.io): It's a PEP 517/PEP 660 compatible
  build backend used by Hatch, a modern, extensible Python project manager.
  It provides a standardized build system with reproducible builds by default,
  robust environment management with support for custom scripts, easy publishing
  to PyPI or other indexes, version management, and configurable project generation
  with sane defaults. Hatchling ensures that your builds are reproducible, so you
  can be confident that they will always produce the same results. It also helps
  you manage your Python environments, so you can be sure that your projects have
  the correct dependencies.
```

Reply:
I think this should also mention that hatchling should ideally be used only for pure python projects